wiki/framework/cargo-framework.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: wiki
+title: Cargo Framework
+description: Explains how to set-up the cargo framework for custom objects and vehicles.
+group: framework
+order: 5
+parent: wiki
+---
+
+## 1. Config Values
+
+### 1.1 Preparing a vehicle for ACE3 cargo
+
+```c++
+class CfgVehicles {
+    class yourVehicleBaseClass {
+        ace_cargo_space = 4;  // The cargo space your vehicle has
+        ace_cargo_hasCargo = 1;  // Enables cargo to be loaded inside the vehicle (1 = yes, 0 = no, default = 1)
```

Comment:
Everywhere else the style used is: `(Optional) Text (0-no, 1-yes)`

Here I'd just remove the optional/default note though, if a developer doesn't inherit from a class that has this config entry set then that won't be true. It's more of a general inheritance guide then, no need to be here.